images/prune-ocp-projects/buildah.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Create the container
+container=$(buildah from registry.access.redhat.com/ubi8/ubi)
+buildah config --label io.k8s.description="OCP Project Pruner" --label io.k8s.display-name="OCP Project Pruner" --env PATH='$PATH:/usr/local/bin' $container
+buildah add $container include/prune-ocp-projects.sh /usr/local/bin/
```

Comment:
@pabrahamsson Looks like we need to bump the oc version here as well.

Reply:
Well spotted, I've updated to 4.4